Gemfile

```diff
@@ -12,7 +12,12 @@ end
 
 gemspec
 
-gem 'rake'
+if RUBY_VERSION < '1.9.3'
```

Comment:
My 5pc: There are still many Puppet 2.6, 2.7 on ruby 1.8 installations around the planet. Keeping at least rspec-puppet and puppet-lint with ruby 1.8 support will allow these people to easier migrate to newer versions of puppet. YMMV.
I suppose that dropping ruby 1.8 would require a major version bump.